core/templates/dev/head/pages/exploration_editor/ExplorationStatesService.js

```diff
@@ -235,6 +235,8 @@ oppia.factory('ExplorationStatesService', [
       },
       saveInteractionAnswerGroups: function(stateName, newAnswerGroups) {
         saveStateProperty(stateName, 'answer_groups', newAnswerGroups);
+        $rootScope.$broadcast(
+          'saveInteractionAnswerGroups', {state_name: stateName});
```

Comment:
This reads like a directive to save the interaction answer groups. Can we rename to interactionAnswerGroupsSaved or similar?

Also please extract string into named constant and use the latter in multiple places as needed.

Ditto elsewhere.

Reply:
Done.